MongoBundle/Resources/config/metadata_search_reader.yml

```diff
@@ -49,7 +49,7 @@ services:
     open_orchestra.annotation_search.driver_chain:
         class: %open_orchestra.annotation_search.driver_chain.class%
         arguments:
-            - [@open_orchestra.annotation_search.annotation_driver, @open_orchestra.annotation_search.yaml_driver, @open_orchestra.annotation_search.xml_driver]
+            - [@open_orchestra.annotation_search.yaml_driver, @open_orchestra.annotation_search.xml_driver, @open_orchestra.annotation_search.annotation_driver]
```

Comment:
I think that you should use the caller addDriver from the class instead of giving all arguments to the constructor